### DISCUSSION TITLE
Import AWS Lmabda Layer in Github actions

### DISCUSSION THREAD

**Discussion by HarshShahCitrusbug** [aws/aws-cdk#27141] [Q&A]
I have setup the Layer file in AWS Lambda. And its working quite fine with Lambda.
I wrote test cases for functionality coverage and that test cases not able to Import AWS Layer in CI-CD pipeline and due to that test cases
are getting failed.

So is there any solution to Import AWS Layer Github workflow for Testcases?

**Comment by mmartinez-maxi** [comment]
I have the same problem, my api tests use a layer, but this test cases not able to import the AWS Layer in pipeline...
